docs/maintain-guides-how-to-chill.md

```diff
@@ -0,0 +1,32 @@
+---
+id: maintain-guides-how-to-chill
+title: Chilling
+sidebar_label: Chilling
+---
+
+Stakers can assume any one of the three states: validating, nominating, or chilling. When a staker
+does not want to actively engage in staking - but does not want to unbond their funds either - they
+can choose to temporarily become inactive while keeping their funds staked. The act of stopping
+either nominating or validating is known as _chilling_.
+
+An account can step back from participating in active staking by calling the `chill` extrinsic in
+the [staking pallet][chill extrinsic]. When an account chooses to chill, they will become inactive
+in the next era. The call must be signed by the _controller_ account, not the _stash_.
```

Comment:
Why must the call be from the controller account and not either the controller or stash account?

Reply:
Ah! That makes sense, I totally forgot about the roles there. Maybe @lsaether, do you want to add in a reminder or possibly a link to the accounts page about the controller and stash roles?

Reply:
Great idea.